### ISSUE TITLE
Error: Can't resolve '@babel/runtime/regenerator'

### ISSUE THREAD

**Issue by vlandham** [afitiskin/redux-saga-routines#53]
I believe the `@babel` dependencies for redux-saga-routines may need to be updated. 
Using redux : 4.0.1 and redux-saga: 0.16.2 and redux-saga-routines: 3.1.1, I'm getting an error:

```
./node_modules/redux-saga-routines/es/routinePromiseWatcherSaga.js
Module not found: Can't resolve '@babel/runtime/regenerator' in '/Users/jvallandingham/code/z/strain-brain/node_modules/redux-saga-routines/es'
```

It appears that there may have been a problem with v7.1.1 of babel: (https://github.com/babel/babel/releases/tag/v7.1.1) 

> EDIT: had a publish issue here as well where it compiled against Node 8 instead of Node 6 so 7.1.2 will fix this.
> Also force publish @babel/runtime and @babel/runtime-corejs2.

If i add `@babel/runtime` version 7.1.2 to my project, this issue goes away - and things appear to work as expected.

**Comment by afitiskin** [comment]
I've published [version 3.1.2](https://github.com/afitiskin/redux-saga-routines/releases/tag/3.1.2) with updated babel deps, please check if it works well for you

**Comment by vlandham** [comment]
Thanks for taking a look! I think the issue is fixed by your peer dependency in 3.1.3. 
I created a minimal example of how i'm using redux-saga-routines here: https://github.com/vlandham/redux_saga_routines_test to test. I believe the new (2.0) version of create-react-app provides the babel peer dependency package. Where I was having troubles before, I am using the 1x version of create-react-app. Adding the runtime peer dependency solved the import issue. 

Thanks for such a useful package!
